rllib/BUILD

```diff
@@ -2732,6 +2732,15 @@ py_test(
 #    args = ["--enable-new-api-stack", "--num-agents=2", "--as-test", "--framework=torch", "--stop-reward=-100.0", "--num-cpus=4"],
 # )
 
+py_test(
```

Comment:
Ah, good point. I was thinking we should add this under `examples/checkpoints`.
What do you think?